client/src/components/BirthdayIconInput.js

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+import Input from './Input.js';
+
+export const BirthdayIconInput = styled(Input)`
+  placeholder: none;
+  background-image: url('data:image/svg+xml,%3Csvg%20aria-hidden%3D%22true%22%20focusable%3D%22false%22%20data-prefix%3D%22fas%22%20data-icon%3D%22birthday-cake%22%20class%3D%22svg-inline--fa%20fa-birthday-cake%20fa-w-14%22%20role%3D%22img%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20448%20512%22%3E%3Cpath%20fill%3D%22%23FEA630%22%20d%3D%22M448%20384c-28.02%200-31.26-32-74.5-32-43.43%200-46.825%2032-74.75%2032-27.695%200-31.454-32-74.75-32-42.842%200-47.218%2032-74.5%2032-28.148%200-31.202-32-74.75-32-43.547%200-46.653%2032-74.75%2032v-80c0-26.5%2021.5-48%2048-48h16V112h64v144h64V112h64v144h64V112h64v144h16c26.5%200%2048%2021.5%2048%2048v80zm0%20128H0v-96c43.356%200%2046.767-32%2074.75-32%2027.951%200%2031.253%2032%2074.75%2032%2042.843%200%2047.217-32%2074.5-32%2028.148%200%2031.201%2032%2074.75%2032%2043.357%200%2046.767-32%2074.75-32%2027.488%200%2031.252%2032%2074.5%2032v96zM96%2096c-17.75%200-32-14.25-32-32%200-31%2032-23%2032-64%2012%200%2032%2029.5%2032%2056s-14.25%2040-32%2040zm128%200c-17.75%200-32-14.25-32-32%200-31%2032-23%2032-64%2012%200%2032%2029.5%2032%2056s-14.25%2040-32%2040zm128%200c-17.75%200-32-14.25-32-32%200-31%2032-23%2032-64%2012%200%2032%2029.5%2032%2056s-14.25%2040-32%2040z%22%3E%3C%2Fpath%3E%3C%2Fsvg%3E');
```

Comment:
This is an inline-SVG. 
Instead, you could save the SVG in a file in /assets like the other SVG files and load it:

```
import birthdayIcon from '../assets/birthday-icon.svg';

const BirtdayInput = styled(Input)`
  background-image: url(${birthdayIcon});
  ....
`;


```